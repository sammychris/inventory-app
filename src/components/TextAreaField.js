import React from 'react';
import styled from 'styled-components';


const TextAreaField = ({value, event}) => {
    return (
      <Textarea value={value} onChange={event} />
    );
  }


export default TextAreaField;


const Textarea = styled.textarea`
  width: 500px;
  height: 100px;
  font-size: 18px;
  margin: 5px 0;
`;
