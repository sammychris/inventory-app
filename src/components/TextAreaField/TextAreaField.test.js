import { shallow } from "enzyme";
import TextAreaField from "./TextAreaField";

const wrapper = shallow(<TextAreaField />);

test("expect to render a TextAreaField", () => {
  expect(wrapper.length).toEqual(1);
});

test("expect to make a snap shot to TextAreaField component", () => {
  expect(wrapper).toMatchSnapshot();
});