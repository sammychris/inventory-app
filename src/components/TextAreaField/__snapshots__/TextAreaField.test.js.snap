// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snap shot to TextAreaField component 1`] = `ShallowWrapper {}`;
